CPPLINT.cfg

```diff
@@ -8,6 +9,7 @@ filter=-legal/copyright
 filter=-build/include
 filter=-build/include_what_you_use
 filter=-build/include_order
+filter=-build/header_guard
```

Comment:
Do you have an idea why this rule was deactivated?

Reply:
If I remember correctly Frederic had trouble with this rule on Windows because the path in cpplint was not starting from the same root as in Linux so header guards were wrong...

Reply:
OK. We should think about a way to deactivate this only on WIndows to make sure our headers are consistent in linux, but it can be addressed in another commit